packages/block-editor/src/hooks/font-size.js

```diff
@@ -136,6 +137,54 @@ export function FontSizeEdit( props ) {
 	);
 }
 
+/**
+ * Add inline styles for font sizes.
+ * Ideally, this is not needed and themes load the font-size classes on the editor.
+ *
+ * @param  {Function} BlockListBlock Original component
+ * @return {Function}                Wrapped component
+ */
+const withFontSizeInlineStyles = createHigherOrderComponent(
+	( BlockListBlock ) => ( props ) => {
+		const {
+			name: blockName,
+			attributes: { fontSize, style },
+			wrapperProps,
+		} = props;
+
+		const { fontSizes } = useSelect( ( select ) =>
+			select( 'core/block-editor' ).getSettings()
+		);
+
+		// Return early if the block doesn't allow modify the font-size,
+		// already has a inline font-size,
+		// or doesn't have a class to extract font-size from.
+		if (
+			! hasBlockSupport( blockName, FONT_SIZE_SUPPORT_KEY ) ||
+			style?.typography?.fontSize ||
+			! fontSize
```

Comment:
We shouldn't return early if the font size is not defined because it remounts the whole block when you select aa font size closing the inspector panels.

Reply:
I haven't seen that :thinking: 

![2005-27-font-size-filter](https://user-images.githubusercontent.com/583546/83008457-753e5c80-a015-11ea-8836-64faa8d0afeb.gif)

Reply:
Maybe because the typography panel is open by default. if we change it to be closed by default you'll notice the issue. I had the issue with the color panel.

Reply:
I still think we need to remove `style?.typography?.fontSize || ! fontSize` from here and only check the support here before merging.

Reply:
Yes, I'll check it now

Reply:
@youknowriad Do you know what `style?.typography` is? That's not a CSS prop?